content/01.main_content.md

```diff
@@ -1,36 +1,37 @@
+  Additionally, the nominating committee inducted their first China-based fellow [@doi:10.1016/j.cels.2021.07.007].
 
 Audits can also provide evidence for specific actions.
   A recent study of disparities in scientific journalism found that predicted East Asian names were under-quoted and under-mentioned [@doi:10.1101/2021.06.21.449261]. 
-  However, this disparity was almost completely removed when considering quoted or mentioned people with a US-affiliated paper cited in the article.
-  This suggests a regional bias, arguing for more regionally focused or co-located journalists.
+  Interestingly, this disparity differed across the predicted name-origins of journalists.
+  Furthermore, the differences between journalists was almost completely removed when considering people with a US-affiliated paper cited in the article.
+  This suggests a regional bias, arguing for more regionally focused journalists.
 
-Audits of representation would ideally be done with all individuals self-reporting gender, ethnicity, and other identifications [@doi:10.1371/journal.pcbi.1003903].
-  While this may be possible prospectively, surveying for self-identification is impractical for large groups and often impossible retrospectively [@http://mediashift.org/2014/11/how-to-ethically-and-responsibly-identify-gender-in-large-datasets/].
-  Computationally-derived predictions allow for audits on a scale that would not be possible otherwise.
-  Numerous tools exist to algorithmically infer gender, nationality, and ethnicity information using only the feature most likely to be non-missing in a relevant dataset: an individual's name. [@doi:10.1093/pan/mpw001; @http://genderize.io/]
-  Most of these models are highly scalable, allowing auditors to define the scope of their target group and background population as broadly as needed.
+Ideally, audits would only use self-reported gender, ethnicity, and other identifications [@doi:10.1371/journal.pcbi.1003903].
+  While this is possible prospectively, surveying is impractical for large groups and often impossible retrospectively [@http://mediashift.org/2014/11/how-to-ethically-and-responsibly-identify-gender-in-large-datasets/].
+  In contrast, computationally-derived predictions allow for large-scale audits.
+  Numerous tools exist to algorithmically infer gender, nationality, and ethnicity using the feature likely to be present in most datasets: an individual's name. [@doi:10.1093/pan/mpw001; @http://genderize.io/]
+  These models are highly scalable, allowing auditors to broadly define the scope of their audit.
   
-Prediction models are not a panacea; several factors limit both their accuracy at recapitulating self-reported information and their ability to address the underlying motivating questions of diversity audits.
+Prediction models are not a panacea; several factors limit both their accuracy and utility.
```

Comment:
I'm sad to see this distinction go but I agree it must be done for the word count. My kingdom for a couple extra hundred words to this article...

Reply:
550 is too short! it requires loss of information :(